types/flubber/index.d.ts

```diff
@@ -0,0 +1,29 @@
+export type Interpolator = (t: number) => string
+
+export type Shape = number[][] | string
+
+export interface Options {
+    maxSegmentLength?: number;
+    string?: boolean;
+    single?: boolean;
+}
+
+export function interpolate(fromShape: Shape, toShape: Shape, options?: Options): Interpolator;
+
+export function toCircle(fromShape: Shape, x: number, y: number, r: number): Interpolator;
+
+export function toRect(fromShape: Shape, x: number, y: number, width: number, height: number, options?: Options): Interpolator;
+
+export function fromCircle(x: number, y: number, radius: number, toShape: Shape, options?: Options): Interpolator;
+
+export function fromRect(x: number, y: number, width: number, height: number, toShape:Shape, options?:Options): Interpolator;

```

Comment:
```suggestion
export function fromRect(x: number, y: number, width: number, height: number, toShape: Shape, options?: Options): Interpolator;

```

The lint tests haven't run yet, but these won't pass because some of the spacing is off. Can you fix the spacing?